.gitignore

```diff
@@ -19,3 +19,4 @@ __pycache__/
 robocop.log
 infrastructure_configuration.yaml
 log/
+text-generation-inference/
```

Comment:
curious question: what generates this directory?

Reply:
I think it was generated as part of model serving tests I run on the same env.